packages/pangea-django/pyproject.toml

```diff
@@ -0,0 +1,41 @@
+[tool.poetry]
+name = "pangea-django"
+version = "1.0.0"
+description = "Pangea Authentication Addon for Django"
+authors = ["Govind Salinas <govind.salinas@pangea.cloud>"]
+license = "MIT"
+readme = "README.md"
+homepage = ""
+repository = ""
+keywords = ["Pangea", "Django", "AutnN"]
+classifiers = [
+    "Topic :: Software Development",
+    "Topic :: Software Development :: Libraries",
+]
+packages = [
+    { include = "pangea_django" }
+]
+
+[tool.poetry.dependencies]
+python = "^3.7.2"
+requests = "^2.31.0"
+cryptography = "41.0.5"
+schema = "^0.7.5"
+python-dateutil = "^2.8.2"
+alive-progress = "^2.4.1"
+pydantic = "^1.10.2"
+pytest = "^7.2.0"
+deprecated = "^1.2.13"
+aiohttp = "^3.8.6"
+asyncio = "^3.4.3"
+google-crc32c = "^1.5.0"

```

Comment:
Most of these dependencies look like copy-paste from pangea-sdk; only include those actually needed by the new package.